### ISSUE TITLE
Duplicate output in log file

### ISSUE THREAD

**Issue by JEM-Mosig** [RasaHQ/rasa#4113]
**Rasa version**:
1.1.8, commit e3ae94995d

**Python version**:
3.7 (I have to check once I get home)

**Operating system** (windows, osx, ...):
Observed on Ubuntu and Mac OS X

**Issue**:
Both `rasa train` and `rasa test` tend to write output twice for no reason. This happens on my Ubuntu desktop and on Vova's MacBook, but not on my MacBook.

**Error (including full traceback)**:
There is no error message generated. Here are a few lines of example output:
```
I0726 08:59:29.732669 140506679359296 test.py:537] Evaluation Results on CONVERSATION level:
2019-07-26 08:59:29 INFO     rasa.core.test  - Evaluation Results on CONVERSATION level:
I0726 08:59:29.732778 140506679359296 test.py:539] 	Correct:          83 / 250
2019-07-26 08:59:29 INFO     rasa.core.test  - 	Correct:          83 / 250
```
or
```
I0726 08:06:45.822660 140072724084544 generator.py:318] Finished phase (250 training samples found).
2019-07-26 08:06:45 DEBUG    rasa.core.training.generator  - Finished phase (250 training samples found).
```

**Command or request that led to error**:
```
rasa test core \
	--stories data/train \
	--out results/train \
	-vv &> test-train.out
```
or
```
rasa train core \
	--stories data/train \
	--domain domain.yml \
	--config config.yml \
	--fixed-model-name transformer \
	-vv &> train.out
```

**Comment by Ghostvv** [comment]
I have the same on my Mac:
```
I0726 17:29:45.722398 4602045888 featurizers.py:649] Created 1441 action examples.
2019-07-26 17:29:45 DEBUG    rasa.core.featurizers  - Created 1441 action examples.
I0726 17:29:45.728195 4602045888 memoization.py:160] Memorized 30 unique examples.
2019-07-26 17:29:45 DEBUG    rasa.core.policies.memoization  - Memorized 30 unique examples.
I0726 17:29:48.772770 4602045888 agent.py:797] Persisted model to '/var/folders/k6/6hbz34d90fl1s5ywp6srmsg80000gn/T/tmppk3t7rj2/core'
2019-07-26 17:29:48 INFO     rasa.core.agent  - Persisted model to '/var/folders/k6/6hbz34d90fl1s5ywp6srmsg80000gn/T/tmppk3t7rj2/core'
```

**Comment by Ghostvv** [comment]
@dakshvar22 welcome to the club

**Comment by Ghostvv** [comment]
@tmbo any ideas about this problem?

**Comment by tmbo** [comment]
@Ghostvv I couldn't reproduce this locally when I tried to yesterday. I used the master branch and py 3.6

So the only difference is the python version I guess. @JEM-Mosig can you try this on your machine with a python 3.6 please?

**Comment by Ghostvv** [comment]
@tmbo I think it appears on `refactor-tf` branch

**Comment by erohmensing** [comment]
@dakshvar22 fixed this in 1.3 🚀
